### ISSUE TITLE
Starting Container status page

### ISSUE THREAD

**Issue by GamerClassN7** [vmorganp/Lazytainer#71]
Hello,
Is there any way how to show static html page when starting the container, for example starting the service ?  I love this project but user are sometimes confused and scared by default browse error page. This feature would be really nice if not implemented already (in i didnot found it )

Thank you in advance for information :)

**Comment by vmorganp** [comment]
I agree that such a thing would be nice, however, because Lazytainer services non http traffic as well, it would be difficult (to a degree that I am not willing to do it) to determine how to respond with "hold please" to every service that uses Lazytainer.

Theoretically this would be fairly easy for http based traffic, and you can already get that using these projects:
1. https://github.com/ItsEcholot/ContainerNursery
2. https://github.com/acouvreur/sablier

However just from our own /examples directory, I would not want to manage how to respond that way for:
- Minecraft
- Zerotier
- Satisfactory
- Whatever other services people are using behind Lazytainer not Pr'd here

My best judgement says that this feature would require either immense time and effort on my part, or would remove some of the versatility offered by Lazytainer today, neither of which I want to do.

If you know of a way to make it happen without either of those things, I'm open to it.

---

tldr; If that feature is a deal breaker, you should probably go check one of the linked repos.

**Comment by GamerClassN7** [comment]
Hi i tried both project but i was able to correctly setup only the [Lazytainer](https://github.com/vmorganp/Lazytainer) , i relay like your service agnostic approach which is really beneficial for whole my infrastructure.  This feature would be nice to have for those services running over Http/Https

**Comment by leleogere** [comment]
Wouldn't an additional flag like `lazytainer.group.myservice.serveWaitingPage=true` be possible? Or maybe even `lazytainer.group.myservice.ports.80.serveWaitingPage=true` if it needs to be specified by port. I'm not sure at all if this is possible, as I'm not familiar with docker compose labels and lazytainer internals.
